samples/hello_world/prj.conf

```diff
@@ -1 +1,2 @@
 # nothing here
+CONFIG_SECURE_BOOT=y
```

Comment:
I think this was unintentional; right?

Reply:
Correct, I have to start reviewing my amends a bit better :D